ajax/libs/vue-amap/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "vue-amap",
+  "description": "A GaoDe Map Component Library for Vue.js.",
+  "filename": "index.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ElemeFE/vue-amap.git"
```

Comment:
@maruilian11 can you please just take a look at the repository if there are cases like this before asking?